src/ui/components/Icon/external.svg

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Extra blank line?

Reply:
Ah, just a result of copying from Sketch. I'll remove it 😄

Reply:
also please see `src/ui/components/Icon/fox.svg` and `src/ui/components/Icon/magnifying-glass.svg`